dartsim/src/LinkFeatures_TEST.cc

```diff
@@ -79,16 +81,17 @@ TestWorldPtr LoadWorld(
 {
   sdf::Root root;
   const sdf::Errors errors = root.Load(_sdfFile);
-  EXPECT_TRUE(errors.empty());
+  EXPECT_TRUE(errors.empty()) << errors;
   const sdf::World *sdfWorld = root.WorldByIndex(0);
-  // Make a copy of the world so we can set the gravity property
-  // TODO(addisu) Add a world property feature to set gravity instead of this
-  // hack
-  sdf::World worldCopy;
-  worldCopy.Load(sdfWorld->Element());
-
-  worldCopy.SetGravity(math::eigen3::convert(_gravity));
-  return _engine->ConstructWorld(worldCopy);
+  EXPECT_NE(nullptr, sdfWorld);
+
+  auto graphErrors = sdfWorld->ValidateGraphs();
+  EXPECT_EQ(0u, graphErrors.size()) << graphErrors;
+
+  TestWorldPtr world = _engine->ConstructWorld(*sdfWorld);
+  EXPECT_NE(nullptr, world);
+  world->SetGravity(_gravity);
```

Comment:
Can you add an expectation on `GetGravity` just to exercise the API?

Reply:
I had intended to do that, but then I realized that I would need to move the `AssertVectorApprox` class definition up above this function in order to use it in the expectation. So I stalled out on doing it. I'll do it now though

Reply:
done in https://github.com/ignitionrobotics/ign-physics/pull/275/commits/ef1012d9170f962756d8dfa7668d6a43a099eea9